tests/conftest.py

```diff
@@ -40,6 +58,8 @@ def get_all_classes():
 
 @pytest.fixture()
 def minimal_attribute_dict():
+    """defines minmal attributes (values) needed to initiate each transformer named (key).
+    New transformers need to be added here"""
```

Comment:
S: Could also mention here that these values may need updating when a transformer is added to the new testing framework? (if there's some arg refactoring in the process)

Reply:
Thanks for the suggestion, I think explaining how it works is enough to highlight that this might be an area to debug?  I think this suggestion might be a bit excessive.

Reply:
Yeah that makes sense